caesar_cipher.rb

```diff
@@ -0,0 +1,97 @@
+# Program that will cipher a String that the user will pass to it.
+
+class Caesar
+    ALPHABET_UPPERCASE = Array ('A'..'Z')
+    ALPHABET_LOWERCASE = Array ('a'..'z')
+
+    def initialize(text, num)
+        @text = text
+        @num = num
+    end
+
+    # Method that ciphers the text that we give it
+    def cipher
+        # It will check if there is text to cipher and if there is a valid number to change the letters
+        return puts "The text given is null and/or the number given is 0." if text == "" || num.zero?
+        text_split = text.split('')
+        puts refactor_text(text_split)
```

Comment:
To make the code more readable, it is worth adding one blank line between the `return` method and the business logic.  
```suggestion
        return puts "The text given is null and/or the number given is 0." if text == "" || num.zero?
        
        text_split = text.split('')
        puts refactor_text(text_split)
```

Reply:
+1